### PR TITLE
Fix e2e test for newly created project/namespace

### LINKED ISSUE #12338: Fix `namespace-picker.spec.ts` e2e test failing in 2.10
**Setup**
- Rancher version: 2.10
- Rancher UI Extensions:
- Browser type & version:

**Describe the bug**

After updating our e2e test env pointing to the latest backend(2.10) a test from `namespace-picker.spec.ts` fails.

Test case:
`newly created project/namespace appears in namespace picker`

Path:
`cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts`

**Additional context**

[The PR](https://github.com/rancher/dashboard/pull/12310) to update e2e test env to point to the latest backend.

### PR DESCRIPTION
**Pull request by momesgin**
### Summary
Fixes #12338 

### Technical notes summary

Increasing the timeout and scrolling into view was needed to fix the test.
### Areas or cases that should be tested

n/a
### Areas which could experience regressions

n/a
### Screenshot/Video

n/a
### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Review by richard-cox — DISMISSED** [review]

**Review by eva-vashkevich — APPROVED** [review]
lgtm

File: cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts [hunk 1]
```diff
@@ -191,40 +195,63 @@ describe('Namespace picker', { testIsolation: 'off' }, () => {
...
+        namespacePicker.toggle();
+        cy.contains(projNameToDelete).should('exist').scrollIntoView().and('be.visible');
+        cy.contains(nsNameToDelete).should('exist').scrollIntoView().and('be.visible');
+
+        // delete project and ns
+        cy.deleteRancherResource('v1', 'namespaces', nsNameToDelete);
+        cy.deleteRancherResource('v3', 'projects', projIdToDelete);
+
+        // check ns picker
+        cy.reload();
```
Comment (izaac) [inline cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts:243]: Is the reload necessary ?

Comment (momesgin) [reply cypress/e2e/tests/pages/explorer2/namespace-picker.spec.ts]: I just removed it and it worked without it locally

**Review by izaac — APPROVED** [review]
lgtm

**Review by izaac — APPROVED** [review]
:shipit:

**Pull request merged**